change/@microsoft-teams-js-2982475b-f77f-44fe-8106-e56b04b27ddc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "added size property in openFilePreview API input params",
+  "packageName": "@microsoft/teams-js",
+  "email": "email not defined",
```

Comment:
This seems odd. Is this changelist auto generated? If so, i thought it should set this email.

Reply:
yes it was auto generated.